qemu/tests/multi_nics_stress.py

```diff
@@ -26,16 +26,20 @@ def launch_netperf_client(test, server_ips, netperf_clients, test_option,
             if utils_misc.wait_for(n_client.is_netperf_running, 30, 0, 1,
                                    "Wait netperf test start"):
                 logging.info("Netperf test start successfully.")
-            else:
```

Comment:
Because the netperf command runs in the background, it will output "/var/tmp/netperf-2.7.1/src/netperf -H 10.72.32.223 -l 120 -t TCP_STREAM > /dev/null" when it exits, which will make the program mistakenly believe it is the result of the command "ps -C netperf". Therefore, I think there is no practical significance to judge here.